tests/Zend/Cache/LibmemcachedBackendTest.php

```diff
@@ -42,6 +42,7 @@
 class Zend_Cache_LibmemcachedBackendTest extends Zend_Cache_CommonExtendedBackendTest {
 
     protected $_instance;
+    protected $skipped;
```

Comment:
In some cases I did this to short-circuit the tearDown method.  In this particular case there's a call to `sleep` that would slow this test down quite a bit (tearDown is called regardless if the test was skipped or not).